### ISSUE TITLE
[unrelated] Lovely app

### ISSUE THREAD

**Issue by working-name** [ivoronin/TomatoBar#69]
Hello! Just wanted to say thank you, you did a great job. I love this app.

**Comment by nikitapashinsky** [comment]
Agreed! I feel like TomatoBar is massively underrated. It does exactly one thing, beautifully. Not bloated with dozens of features. Truly a perfect menu bar app. Also, nice touch with the sounds! Thank you @ivoronin!
